source/sdk/math.ooc

```diff
@@ -291,13 +297,18 @@ extend Float {
 		result := coefficient toString() >> "E" & power toString()
 		result
 	}
-	equals: func (other: This, tolerance := 0.0001f) -> Bool { (this - other) abs() < tolerance }
+	equals: func (other: This, tolerance := This defaultTolerance) -> Bool { (this - other) abs() < tolerance }
+	lessThan: func (other: This, tolerance := This defaultTolerance) -> Bool { this - other < tolerance }
+	greaterThan: func (other: This, tolerance := This defaultTolerance) -> Bool { this - other > tolerance }
+	lessOrEqual: func (other: This, tolerance := This defaultTolerance) -> Bool { this equals(other) || this lessThan(other) }
+	greaterOrEqual: func (other: This, tolerance := This defaultTolerance) -> Bool { this equals(other) || this greaterThan(other) }
 }
 
 extend LDouble {
 	pi ::= static 3.14159_26535_89793_23846_26433_83279L
 	e ::= static 2.718281828459045235360287471352662497757247093699959574966L
-	
+	defaultTolerance ::= static 0.0001L
```

Comment:
Can't reuse the value from Double class because rock:
> error Property tries to define a field in type extension.
>
>    defaultTolerance ::= static Double defaultTolerance as This